sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -1,131 +1,3 @@
-
-Pets are represented in the message body as a JSON object:
-
-```json
-{
-    "name": "snoopy",
-    "species": "beagle"
-}
-```
-
-An API using model types could be:
-
-```csharp
-// This is an example model class
-public class Pet
-{
-    string Name { get; }
-    string Species { get; }
-}
-
-Response<Pet> GetPet(string dogName);
-Response SetPet(Pet dog);
-```
-
-While the protocol methods version would be:
-
-```csharp
-// Request: "id" in the context path, like "/pets/{id}"
-// Response: {
-//      "name": "snoopy",
-//      "species": "beagle"
-// }
-Response GetPet(string id, RequestContext context = null)
-// Request: {
-//      "name": "snoopy",
-//      "species": "beagle"
-// }
-// Response: {
-//      "name": "snoopy",
-//      "species": "beagle"
-// }
-Response SetPet(RequestContent requestBody, RequestContext context = null);
-```
-
-**[Note]**: This document is a general quickstart in using SDK Clients that expose '**protocol methods**'.
-
-## Usage
-
-The basic structure of calling protocol methods remains the same as that of convenience methods:
-
-1. [Initialize Your Client](#1-initialize-your-client "Initialize Your Client")
-
-2. [Create and Send a request](#2-create-and-send-a-request "Create and Send a Request")
-
-3. [Handle the Response](#3-handle-the-response "Handle the Response")
-
-### 1. Initialize Your Client
-
-The first step in interacting with a service via protocol methods is to create a client instance.
-
-```csharp
-using System;
-using Azure.Pets;
-using Azure.Core;
-using Azure.Identity;
-
-const string endpoint = "http://localhost:3000";
-var credential = new AzureKeyCredential(/*SERVICE-API-KEY*/);
-var client = new PetStoreClient(new Uri(endpoint), credential, new PetStoreClientOptions());
-```
-
-### 2. Create and Send a Request
-
-Protocol methods need a JSON object of the shape required by the schema of the service.
-
-See the specific service documentation for details, but as an example:
-
-```csharp
-// anonymous class is serialized by System.Text.Json using runtime reflection
-var data = new {
-    name = "snoopy",
-    species = "beagle"
-};
-/*
-{
-    "name": "snoopy",
-    "species": "beagle"
-}
-*/
-client.SetPet(RequestContent.Create(data));
-```
-
-### 3. Handle the Response
-
-Protocol methods all return a `Response` object that contains information returned from the service request.
-
-The most important field on Response contains the REST content returned from the service:
-
-```C# Snippet:GetPetAsync
-Response response = await client.GetPetAsync("snoopy", new RequestContext());
-
-var doc = JsonDocument.Parse(response.Content.ToMemory());
-var name = doc.RootElement.GetProperty("name").GetString();
-```
-
-JSON properties can also be accessed using a dynamic layer.
-
-```C# Snippet:AzureCoreGetDynamicJsonProperty
-Response response = client.GetWidget();
-dynamic widget = response.Content.ToDynamicFromJson();
```

Comment:
Does the new document on Learn discuss DynamicData?  It would be good to ensure that all points discussed in this document are either represented in learn or maintained in documentation in the repo for now.

Reply:
I opened a PR that, among other things, shows how to use dynamic. Take a look at https://github.com/dotnet/docs/pull/42351.

Reply:
Many thanks!  Per offline discussion, it would be cool if that doc could also reflect the "three easy steps" here, but I don't see that as blocking to this PR.